optuna/storages/_rdb/storage.py

```diff
@@ -1356,7 +1356,9 @@ def get_current_version(self) -> str:
     def get_head_version(self) -> str:
 
         script = self._create_alembic_script()
-        return script.get_current_head()
+        current_head = script.get_current_head()
+        assert current_head is not None
```

Comment:
[Notes] In my understanding, `current_head` is never `None` in the Optuna codebase since the [`alembic/versions`](https://github.com/optuna/optuna/tree/master/optuna/storages/_rdb/alembic/versions) directory has version files (i.e., `v2.4.0.a.py`).

- `script` is an instance of [`alembic.script.ScriptDirectory`](https://github.com/sqlalchemy/alembic/blob/67a7b94927fe13aa3e74d9172e07abb1fd148f80/alembic/script/base.py#L52)
- `ScriptDirectory` loads version locations from config ([code](https://github.com/sqlalchemy/alembic/blob/67a7b94927fe13aa3e74d9172e07abb1fd148f80/alembic/script/base.py#L166-L202)). Optuna specifies the [default location](https://github.com/optuna/optuna/blob/23e05dc96f44ada4f81b6a19f71ab2d9dab1fbc7/optuna/storages/_rdb/alembic.ini#L29-L32), i.e., `alembic/versions`
- `version_locations`  are passed to [`revision.RevisionMap.__init__`](https://github.com/sqlalchemy/alembic/blob/67a7b94927fe13aa3e74d9172e07abb1fd148f80/alembic/script/base.py#L90) via [`ScriptDirectory.__init__`](https://github.com/sqlalchemy/alembic/blob/67a7b94927fe13aa3e74d9172e07abb1fd148f80/alembic/script/base.py#L218).
- `RevisionMap._revision_map` parses the `version_locations` and add the revisions to `self.head` [here](https://github.com/sqlalchemy/alembic/blob/67a7b94927fe13aa3e74d9172e07abb1fd148f80/alembic/script/revision.py#L191-L241)
-  The first element of `self.heads` is returned as the current head in [`RevisionMap.get_current_head`](https://github.com/sqlalchemy/alembic/blob/67a7b94927fe13aa3e74d9172e07abb1fd148f80/alembic/script/revision.py#L499)
- `ScriptDirectory.get_current_head` calls `RevisionMap.get_current_head` [here](https://github.com/sqlalchemy/alembic/blob/67a7b94927fe13aa3e74d9172e07abb1fd148f80/alembic/script/base.py#L382).